components/tab-tile/tab-tile.js

```diff
@@ -68,7 +68,7 @@ function setTabActive(clickedTab) {
   if (contentContainers.length > clickedTab.dataset.index) {
     contentContainers[clickedTab.dataset.index].classList.add('msds-tabs__content-container--visible')
   }
-
+  ensureCardsContentMatchingHeights()
```

Comment:
you readded the matching heights for tabs? Is it needed?

Reply:
Yes, it is needed, as it recalculates the height every time a user switch tab. Without it, category cards within tabs won't update their height.

Reply:
oh right we had a chat about that :)